test/test_workflow.c

```diff
@@ -0,0 +1,83 @@
+#include <unity.h>
+#include "cJSON.h"
+#include "simulation_lifecycle/error.h"
+#include "simulation_lifecycle/utils/workflow.h"
+
+void setUp(void) {
+
+}
+
+void tearDown(void) {
+
+}
+
+void test_null_cJSON(void){
+    cJSON *file = NULL;
+
+    file = read_workflow_file("../test/data/workflow/MISSING_WORKFLOW_FILE.json");
```

Comment:
There is an issue with this line, but I could not figure it out. `workflow == NULL` in `validate_workflow ` does not seem to work in this case. When I remove this line, all your tests pass :)

Reply:
Hmm, weird it seemed to work before I made the pull request. I will investigate tomorrow and see if I can find the problem, thank you!